apps/projects/templates/meinberlin_projects/includes/phase_indicator.html

```diff
@@ -32,28 +32,30 @@
     </ul>
 </div>
 {% endif %}
+
 {% endwith %}
 
 {% if not object|is_offlineevent %}
-<div class="phase-info">
-    <div class="l-center-6">
-        {% if active_phase %}
+<div class="l-center-6">
+    {% for phase in project.phases %}
```

Comment:
This loops over all phases in a project. It should be limited to the current module.